models/stg_iterable__campaign_metrics.sql

```diff
@@ -0,0 +1,87 @@
+
+with base as (
```

Comment:
We should re-evaluate if we want to bring this model through our package. The data does not seem to be accurate in the base table and I don't want to bring this through to our final models. A customer can leverage this on their own if they want, but from customer interviews it does not seem beneficial to include. Thoughts?

Reply:
yeah was thinking we should just remove!

Reply:
Great! Let's remove the models and all trace of the campaign_metrics reference (ie. get_columns macro, vars in integration_tests, vars in dbt_project.yml, src and stg yml)